include/mbedtls/mbedtls_config.h

```diff
@@ -2534,16 +2534,40 @@
 #define MBEDTLS_ECJPAKE_C
 
 /**
- * \def MBEDTLS_ECP_C
+ * \def MBEDTLS_ECP_LIGHT
  *
  * Enable the elliptic curve over GF(p) library.
+ * This does not include support for scalar multiplication and functionalities
+ * that depepend on it, which include:
+ * - interruptible/restartable operations
+ * - reading of compressed EC points
```

Comment:
The three bullet points are things that do work with `ECP_LIGHT`, but won't work without it. So, I don't think we need to say anything about them now.

Also, I'm not sure the previous item is worth mentioning: it seems quite obvious that if you don't get scalar multiplication, then you don't get interruptible/restartable scalar multiplication either.